criteria/findability.md

```diff
@@ -0,0 +1,51 @@
+# Findability
+
+## Requirements
+
+* The codebase MUST be findable using a search engine by description of the problem it solves in natural language.
+* The codebase MUST be findable using a search engine by codebase name.
+* Maintainers SHOULD submit the codebase to relevant software catalogs.
+* The codebase SHOULD have a website which describes the problem the codebase solves using the preferred jargon of different potential users of the codebase (including technologists, policy experts and managers).
+* The codebase SHOULD have a unique and persistent identifier, such as a [Wikidata item](https://www.wikidata.org/wiki/Wikidata:Introduction) or [FSF software directory listing](https://directory.fsf.org/wiki/Main_Page), mentioning the major contributors, repository location and website.
+* The codebase SHOULD include a machine-readable metadata description, for example in a [publiccode.yml](https://github.com/publiccodeyml/publiccode.yml) file.
+* There MAY be a dedicated domain name for the codebase.
+* The community MAY present regularly at conferences.
+
+## Why this is important
+
+* Proactiveness is needed; just publishing a codebase and hoping it's found doesn't work.
+* The more findable a codebase is, the more potential new collaborators will find it.
+
+## What this does not do
+
+* Ensure new collaborators or replicators
+
+## How to test
+
+Check that the codebase appears in the first set of results on more than one major search engine when you search for:
+
+* plain problem description
+* technical problem description
+* codebase name
+
+## Policy makers: what you need to do
+
+* Contribute a description of the policy area or problem this codebase acts on or operates.
+* Test your problem description with peers outside of your context who aren't familiar with the codebase.
+* Present on how the codebase implements the policy at relevent conferences.
```

Comment:
Is this for the policy makers? The codebases implement policies made by policy makers, no? So I'd say this is downstream of policy makers and they are responsible for making it findable?

Reply:
We meant this for policy makers, yes. When policymakers (perhaps the CIO, or even the Mayor) of a municipality developing a codebase talk about it proudly, they will probably have an audience that is more likely to be able to make decisions to join the effort. So while the downstream also is important, we wanted to address an opportunity that is often overlooked.